libs/sdk-ui-filters/src/DateFilter/accessibility/keyboardNavigation.ts

```diff
@@ -0,0 +1,238 @@
+// (C) 2025 GoodData Corporation
```

Comment:
Is this copied from somewhere? This seems like a generic logic that just have specific IDs and date filter naming inside, otherwise it could be used by any component? Can this be used by some other component and does it make sense to make this shared in sdk ui kit?

Reply:
No, created this from scratch customized for this specific Date Filter use cases, I do not believe it can be shared with other, since is handling specific date filters keyboard navigation scenarios.